### PR TITLE
v2v debug code

### PR DESCRIPTION
**Pull request by xiaodwan**
Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

File: v2v/tests/src/v2v_options.py [hunk 1]
```diff
@@ -74,6 +74,15 @@ def run(test, params, env):
     error_flag = 'strict'
     estimate_file = ''
 
+    def debug_func():
+        try:
+            pwd.getpwnam(v2v_user)
+            cmd = su_cmd + "'virsh pool-list --all'"
```
Comment (yantian0) [inline v2v/tests/src/v2v_options.py:80]: Here only debug for 'virsh pool-list' command. How about make cmd as the parameter, so other command could invoke this debug function?

Comment (xiaodwan) [reply v2v/tests/src/v2v_options.py:80]: This is only a pr for debugging problems, it will be closed or be changed to fix real problems.

Comment (yantian0) [reply v2v/tests/src/v2v_options.py:80]: ok, got it.

**Comment by chunfuwen** [comment]
Please resolve code conflict

**Pull request merged**